src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -370,6 +379,7 @@ function BaseTextInput(
                                 selection={inputProps.selection}
                                 readOnly={isReadOnly}
                                 defaultValue={defaultValue}
+                                markdownStyle={markdownStyle}
```

Comment:
Markdown does not grow correctly in room descriptions resulting in half cut lines, ref: https://github.com/Expensify/App/issues/48281